### PR TITLE
🎨🔥🚨📦💬 change linter, add lint-related changes

### PR DESCRIPTION
**Pull request by bommezijn**
Hi,

I've updated the development branch to use eslint-google and edited server.js to be free of linting errors. This mostly consisted of the following;

adding JSDOC comments
removing or formatting lines longer than 80 characters
other stuff that is auto-fixable by using the shortcut: ⌘ · or ctrl ·

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -140,12 +146,17 @@ function volgendeFilm(req, res) {
 
 app.post('/volgendeFilm-succes', urlencodedParser, volgendeFilm);
 
-
+/**
+ * @title succes refresh?
```
Comment (bommezijn) [inline server.js:150]: Asked a question about server.js @150

**Comment by bommezijn** [comment]
To be able to commit this PR, two people need to approve this code review.

Approving a code review can be done by going to the commit (above this page) click on view changes
review it, comment, or request a change in the code.

If changes are acceptable in your opinion, go to the top of the page click review changes
select approve, type a message, and click submit review.

**Review by ParvinBDJ — APPROVED** [review]
The code looks clean. I like that airbnb is gone.

**Review by uxtothemax — APPROVED** [review]
Looks good!

**Pull request merged**